CVLByExample/LastReverted/README.md

```diff
@@ -0,0 +1,37 @@
+# Last Reverted
+
+Sometimes in our verification we'll want to take note of whether a call was reverted.
+In this example we have a minimal implementation of ERC20 in `ERC20.sol` and we want to verify the conditions that makes the `transfer` and `approve` functions revert. In other words we want to verify that these functions will revert if and only if certain conditions are met and never in any other case (maybe excluding some invalid states). We can achieve this by using `@withrevert` and `lastReverted`.
+
+Normally when a call reverts the prover will ignore the path the led to this call. We can signal the prover to not do that by calling the function while adding `@withrevert`.
```

Comment:
Normally when a call reverts, the prover will ignore the path that led to this call

Reply:
As default, the Prover ignores reverting paths.